rs/bitcoin/consensus/src/payload_builder.rs

```diff
@@ -4,10 +4,8 @@ mod parse;
 #[cfg(test)]
 mod tests;
 
-// TODO: This module does not compile, and was not caught by the compiler
-// as it was behind a non-existent feature flag.
-// #[cfg(test)]
```

Comment:
fyi: I changed this back to
`// #[cfg(all(test, feature = "proptest"))]`

Reply:
Not sure what you mean. I still see it commented out on master. Does it work if we use feature = "proptest"? I thought it was a non-existent feature so the code was actually never compiled.

Reply:
Yes, it is still commented out, but I left the cfg macro as it was `#[cfg(all(test, feature = "proptest"))]` instead of changing it to `#[cfg(test)]`.

We have other crates that have added the feature `proptest` and feature gate modules based on it. See [example here](https://sourcegraph.com/github.com/dfinity/ic@master/-/blob/rs/consensus/src/consensus.rs?L23%3A1-25%3A1).

What I assume would be the correct way to fix it is adding the `proptest` feature to both `BUILD.bazel` and `Cargo.toml` for this crate.

Reply:
Ack, thanks for the explanation, that makes sense. Let me try to update the code here following the example you mentioned.